superset/examples/misc_dashboard.py

```diff
@@ -211,11 +210,7 @@ def load_misc_dashboard() -> None:
     """
     )
     pos = json.loads(js)
-    slices = (
-        db.session.query(Slice).filter(Slice.slice_name.in_(misc_dash_slices)).all()
-    )
-    slices = sorted(slices, key=lambda x: x.id)
-    update_slice_ids(pos, slices)
+    pos, slices = update_slice_ids(pos)
```

Comment:
Shouldn’t this be `slices = …`?

Reply:
Ah, right! I'm surprised test cases didn't catch this.